openmc/tallies.py

```diff
@@ -1410,6 +1432,765 @@ def export_results(self, filename='tally-results', directory='.',
+                raise ValueError(msg)
+
+            self._outer_product(other, new_tally, binary_op='*')
+            data = self._align_tally_data(other)
+            self_rel_err = data['self']['std. dev.'] / data['self']['mean']
+            other_rel_err = data['other']['std. dev.'] / data['other']['mean']
+            new_tally._mean = data['self']['mean'] * data['other']['mean']
+            new_tally._std_dev = np.abs(new_tally.mean) * \
+                                 np.sqrt(self_rel_err**2 + other_rel_err**2)
+
+        elif isinstance(other, (Integral, Rational)):
+
+            new_tally.name = self.name
+            new_tally._mean = self._mean * other
+            new_tally._std_dev = self._std_dev * np.abs(other)
+            new_tally.estimator = self.estimator
+            new_tally.with_summary = self.with_summary
+            new_tally.num_realization = self.num_realizations
+            new_tally.num_score_bins = self.num_score_bins
+
+            for filter in self.filters:
+                new_tally.add_filter(filter)
+            for nuclide in self.nuclides:
+                new_tally.add_nuclide(nuclide)
+            for score in self.scores:
+                new_tally.add_score(score)
+
+        else:
+            msg = 'Unable to multiply Tally ID={1} ' \
+                  'by {0}'.format(self.id, other)
+            raise ValueError(msg)
+
+        return new_tally
+
+    def __div__(self, other):
+        """Divides this tally by another tally or scalar value.
+
+        This method builds a new tally with data that is the dividend of
+        this tally's data and that from the other tally or scalar value. If the
+        filters, scores and nuclides in the two tallies are not the same, then
+        they are combined in all possible ways in the new derived tally.
+
+        Uncertainty propagation is used to compute the standard deviation
+        for the new tally's data. It is important to note that this makes
+        the assumption that the tally data is independently distributed.
+        In most use cases, this is *not* true and may lead to under-prediction
+        of the uncertainty. The uncertainty propagation model is from the
+        following source:
+
+        https://en.wikipedia.org/wiki/Propagation_of_uncertainty
+
+        Parameters
+        ----------
+        other : Tally or Integer or Rational
+            The tally or scalar value to divide this tally by
+
+        Returns
+        -------
+        Tally
+            A new derived tally which is the dividend of this tally and the
+            other tally or scalar value in the division.
+
+        """
+
+        # Check that results have been read
+        if self.sum is None:
+            msg = 'Unable to use tally arithmetic with Tally ID={0} ' \
+                  'since it does not contain any results.'.format(self.id)
+            raise ValueError(msg)
+
+        new_tally = Tally(name='derived')
```

Comment:
Why is `name='derived'` only given in `__div__` and `__pow__` but not the others?